script/Ciel/Day 13/13_01_01_CIEL13_4_1.txt

```diff
@@ -171,17 +171,17 @@ With a sound of a footstep, a shadow leans over me.
 [sha:d5c7ac5cb728f91af59259037a1cdcea1b30a0ec]{
 -- Page 364, Offset 34303.
 -- なぜか、安堵するような声で彼女は言った。
-For some reason, she speaks in a relieved voice.
+For some reason, she sounds somewhat relieved.
 }
 [sha:3430d1a97edf72d03804a82f62a69b116d80e97b]{
 -- Page 365, Offset 34304. Mods: @n.
 -- 「良かった。これでお互いに罪はありません。
-"That's great. With this, neither of us are at sin.
+"That's good. With this, neither of us will have to bear any sin.
 }
 [sha:be4616049925acb52ce36b7ac9f2a64f5c5c0046]{
 -- Page 365, Offset 34305.
 -- 互いに殺し合うという条件なら、私たちの罪は相殺されます。あとは、生き残った者に罰が残るだけです」
-If we're both trying to kill each other, then our sins will be counteracted. After that, all that will remain is the retribution of the survivor."
+Now that we're both trying to kill each other, our sins will balance each other out. In the end, all that will remain is penance for the survivor."
```

Comment:
now i'm no catholic, but i think you might want to email the pope rq about that logic and get his take

Reply:
i'm a scientologist so i cant email the pope

i CAN email tom cruise though